src/nitclk/examples/nitclk_master_trigger.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import niscope
+import nitclk
+import sys
+
+
+def example(resource_name1, resource_name2, channels, options, length, voltage):
```

Comment:
I don't think it's worth complicating the example with channels.
For example:
* It's not clear that you mean channel N from each device, not multiple channels from the devices.
* `channels` is already used as a member of the session. It works out but can be confusing.
I think just make this fetch from channel 0 on each device.

Reply:
What could be useful, in my opinion, is to make the `resource_name` be an iterable (such as a list) so that this works with N digitizers out of the box.

If you think it's a good idea and the code doesn't get messy. I don't think it will.

Reply:
@marcoskirsch Agreed on the channels part.

In the interest of time, I don't think I'll change the resource name to be a list on this pass, but I'll come in and change that later or do it in a future example.